pallets/xcm-transactor/src/lib.rs

```diff
@@ -632,40 +730,110 @@ pub mod pallet {
 			});
 			Ok(())
 		}
-	}
 
-	impl<T: Config> Pallet<T> {
-		fn transact_in_dest_chain_asset_non_signed(
-			dest: MultiLocation,
-			fee_payer: T::AccountId,
-			fee_location: MultiLocation,
-			call: Vec<u8>,
-			origin_kind: OriginKind,
-
-			fee_amount: Option<u128>,
+		/// Manage HRMP operations
+		#[pallet::weight(0)]
+		pub fn hrmp_manange(
+			origin: OriginFor<T>,
+			action: HrmpOperation,
+			// fee to be used
+			fee: CurrencyPayment<CurrencyIdOf<T>>,
+			// weight information to be used
 			weight_info: TransactWeights,
 		) -> DispatchResult {
+			// WithdrawAsset
+			// BuyExecution
+			// SetAppendix(RefundSurplus, DepositAsset(sov account))
+			// Transact
```

Comment:
what is this comment for? seems like a note to self perhaps

Reply:
Just for informative purposes. LIke this is what we are going to send. Maybe I should add this sentence before the instructions